frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponseNavbar.tsx

```diff
@@ -130,10 +141,22 @@ export const IndividualResponseNavbar = (): JSX.Element => {
       </Flex>
       <Flex gridArea="respondent" justify="center" align="center">
         <Skeleton isLoaded={!isLoading}>
-          <Text textStyle="h2" as="h2">
-            {t('features.common.response')}
-            {currentResponseNumber ? ` #${currentResponseNumber}` : ''}
-          </Text>
+          <Stack direction="row" justify="center" align="center">
+            <Text textStyle="h2" as="h2">
+              {t('features.common.response')}
+              {currentResponseNumber ? ` #${currentResponseNumber}` : ''}
+            </Text>
+            {user?.betaFlags?.signatureField && isSignatureFieldEnabled ? (
```

Comment:
Question: would it make sense to enable the print button for everyone instead of signature enabled admins? Or is this part of data gathering?

Reply:
I think we should aim to have the "Print" button available for everyone in the long term, yes!

A couple of considerations that come to mind:
1. This "Print" button uses the browser's functionality, so the layout and content will differ from the PDF respondent copy.
2. In the long term, we should probably offload all PDF generation (respondent copy, admin copy, etc.) to the server-side rendered / generated PDFs (whether serverless or containerised)

Perhaps for consistency we can feature flag the admin's "Print" button separately so that we can enable / disable it independently of the signature field in user testing? This might help to reduce admin confusion.